docs/dev/annual_updates.rst

```diff
@@ -224,8 +293,8 @@ inconsistencies in the harvested values. See: :doc:`run_the_etl` for more detail
 you'll need to use the ``--ignore-foreign-key-constraints`` argument because new plants
 and utilities probably need to be mapped (read on into next section).
 
-7. Integration Between Datasets
--------------------------------
+7. Integrate Datasets
+---------------------
```

Comment:
I personally think the previous header here was more descriptive. "integrate" could mean a lot of things, but this section is talking about the glue that ties datasets together.

Reply:
I wanted to keep all the headers in the same tense / lead with a verb. I see what you mean, but I think it's probably fine as it will only really be used internally.

Reply:
If your trying to keep the tense the same then can we just say "Connect Datasets" or something?

Reply:
Ok that works!